sycl/unittests/SYCL2020/GetNativeOpenCL.cpp

```diff
@@ -13,9 +11,12 @@
 #include <CL/sycl.hpp>
 #include <CL/sycl/backend/opencl.hpp>
 #include <detail/context_impl.hpp>
-#include <gtest/gtest.h>
+
+#include <helpers/CommonRedefinitions.hpp>
 #include <helpers/PiMock.hpp>
 
+#include <gtest/gtest.h>
+
 #include <iostream>
 #include <memory>
```

Comment:
For `sycl::program` usages.

```suggestion
#include <memory>

#define SYCL2020_DISABLE_DEPRECATION_WARNINGS
```

Reply:
This is SYCL2020 test, so there must be no `sycl::program` usages. Right?
I suggest we do not use deprecated API in our unittests and have dedicated set of tests to validate that deprecated API is supported until we decide to drop such support. Does it make sense?

Reply:
It does

Reply:
Let's address both your comments in a separate PR, if that's okay.